### DISCUSSION TITLE
Possible on two devices simultaneously?

### DISCUSSION THREAD

**Discussion by nkwrig001** [Authenticator-Extension/Authenticator#750] [General]
This might be a dumb question but I'm looking for a way to 'backup' my Google Authenticator codes in the event I ever lose my phone etc.

Is it possible to have to codes function simultaneously from my phone and here in the browser extension?  Or is this just an either or scenario?

Thanks!

**Comment by mymindstorm** [comment]
Yes, you can view the QR code to add the account to a phone by clicking the hidden QR button at the top right of an account.

**Reply by nkwrig001** [reply]
but if the accounts are originally created on my mobile Google Authenticator, can they also be backed up online via the Authenicator extension?

What I'm asking, is can they both operate simultaneously?  I am not trying to remove them from GA, I am simply trying to find a secondary backup option in the event I lose my phone, etc. I can recover my accounts later.

**Reply by mymindstorm** [reply]
Yes, the numbers that show up are a function of the data in the QR code and your computer time.

**Reply by nkwrig001** [reply]
I got it to work, thank you so much!

**Comment by godspower77** [comment]
xpub6CtduFeKXzHJomfXJcS2pF5Z4y5Kyxk7U2anUDhK2ifL32sHGmjhTgnK5ve382wcpd9mtrGxPq8Eb4EAwb4K5eK1UrQeXLBuG95ijsxrRcX

**Comment by godspower77** [comment]
Blockchain wellet 💯 xpub6CtduFeKXzHJomfXJcS2pF5Z4y5Kyxk7U2anUDhK2ifL32sHGmjhTgnK5ve382wcpd9mtrGxPq8Eb4EAwb4K5eK1UrQeXLBuG95ijsxrRcX
